tests/Mono.Android-Tests/Runtime-Microsoft.Android.Sdk/Mono.Android.NET-Tests.csproj

```diff
@@ -45,6 +45,7 @@
   <ItemGroup>
     <TrimmerRootAssembly Include="Java.Interop-Tests" RootMode="All" />
     <_AndroidRemapMembers Include="Remaps.xml" />
+    <_AndroidRemapMembers Include="IsAssignableFromRemaps.xml" Condition=" '$(_AndroidIsAssignableFromCheck)' == 'false' " />
```

Comment:
`IsAssignableFromRemaps.xml` is in its own file, because we have a couple tests that need to use the *real* Android LayoutInflater.

We run only the `Intune` category under `-p:_AndroidIsAssignableFromCheck=false`.